report.xml

```diff
@@ -0,0 +1,5561 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a temporal file that it is not needed. You can remove it from the commits.